### ISSUE TITLE
CSS issues at multiple places, body width expanding more than vw.

### ISSUE THREAD

**Issue by blackwolf08** [CircuitVerse/CircuitVerse#192]
#### I am submitting a -

**Bug Report**

**Would like to SELF ASSIGN the issue**

#### Describe your bug/feature -
Issue with the CSS at index and projet subpages in mobile view.

#### Expected behaviour -
1. Navbar and paginator should not occupy more than 100% of view width.
2.  Stars and views sub tab should not extend beyond 100% of the width in mobile view.
3. (Proposal) Search bar in the navigation pane should be at the top conventionally

#### Current behaviour -

fixed all the issue plus gave some custom propeties to paginaton.

#### Screenshots -

![img_1621](https://user-images.githubusercontent.com/31814079/53752994-25d2b480-3ed6-11e9-96e2-cf1160fa5fa1.jpg)

![img_1620](https://user-images.githubusercontent.com/31814079/53753104-73e7b800-3ed6-11e9-9fae-1e3f8fca3f73.jpg)

![img_1619](https://user-images.githubusercontent.com/31814079/53753035-37b45780-3ed6-11e9-8cc0-ebf63bcdefd0.jpg)

#### Are you working on the issue?

Yes

**Comment by blackwolf08** [comment]
created a pull request to address the issues 1 and 2 and created another pull request to apply my proposed change of search bar to the top of stack :)

**Comment by agarwalrounak** [comment]
I see that you have modified the issue template. Please use it. It helps maintains a uniformity across all issues.

**Comment by blackwolf08** [comment]
thanks @agarwalrounak I have updated the issue, thanks :)

**Comment by satu0king** [comment]
![image](https://user-images.githubusercontent.com/2092958/53860407-e952a600-4006-11e9-8202-8e3949213f8c.png)
This doesn't look very neat. I know if the circuit name is too long, the original CSS breaks and we need to fix that.

**Comment by blackwolf08** [comment]
I’ll work around with the icons then i suppose

**Comment by blackwolf08** [comment]
We can also set the overflow property on the text

**Comment by kunalvishnoi** [comment]
Can i go for the issue as it is still open and there is no update from long time?

**Comment by armudgal** [comment]
@kunalvishnoi Please go forward.

**Comment by woswos** [comment]
I discovered a few more CSS related issues as well. I don't know if I should open a new issue. I will just comment here.

**Describe the bug**

- The footer is shifted to the right in the "Editing Project" page. I tried with Google Chrome & Firefox gives the same results.

Chrome:
![Screen Shot 2019-04-06 at 1 11 07 AM](https://user-images.githubusercontent.com/10498874/55656705-0a2a3900-5809-11e9-8d73-19c069d05d0e.png)

Firefox:
![Screen Shot 2019-04-06 at 1 11 23 AM](https://user-images.githubusercontent.com/10498874/55656712-10201a00-5809-11e9-90be-195ee1c9ca7d.png)

- Logos in the home page are not proportionally sized and they look warped. Especially DigitalOcean logo.

- This one is not really CSS related but the placeholder logo for profile is a low quality photo and can be replaced with an SVG file for decreasing the size and increasing the quality.

**Are you working on this issue? (Yes/No)**
Yes
